mlfinlab/portfolio_optimization/cla.py

```diff
@@ -235,7 +237,7 @@ def _purge_num_err(self, tol):
             if abs(sum(self.weights[index_1]) - 1) > tol:
                 flag = True
             else:
-                for index_2 in range(self.weights[index_1].shape[0]):
+                for index_2 in range(len(self.weights[index_1])):
```

Comment:
Can we improve index_1, index_2 naming?

Reply:
@proskurin They are just indexes for inner and outer loops. I dont know what other names can I give to them